### DISCUSSION TITLE
How to create an optional param in nested routes in Nuxt3 ?

### DISCUSSION THREAD

**Discussion by amanuela97** [nuxt/nuxt#16732] [Questions]
if i navigate to `/services` or `/services/:location/:type` i want to render the same page in both cases. Except in the second case i would want access to $route.param.location & $route.param.type. How can i achieve this ?
The documentation says you need double square brackets to create optional params but it doesnt seem to work in nested routes.

**Comment by danielroe** [comment]
Seems to work fine to me: https://stackblitz.com/edit/github-giqghp?file=pages%2Fservices%2F[[location]]%2F[[type]].vue,app.vue

**Reply by winterjulian** [reply]
Your stackblitz example isn't working. 
[Vue Router warn]: No match found for location with path "/".

**Reply by danielroe** [reply]
It is working fine. You are probably trying to load the index page, which doesn't exist. Instead, try visiting the route requested in the discussion above.
